File: Java/Baju.java
class Baju extends Pakaian{
    private String warna;
    private String tipe;

    Baju(){

    }

    Baju(String warna, String tipe){
        this.warna = warna;
        this.tipe = tipe;
    }

    public void setwarna(String warna){
        this.warna = warna;
    }

    public void settipe(String tipe){
        this.tipe = tipe;
    }

    public String getwarna(){
        return this.warna;
    }

    public String gettipe(){
        return this.tipe;
    }
} 

File: PHP/Baju.php
<?php

require ('Pakaian.php');

class Baju extends Pakaian{

    private $warna = '';
    private $tipe = '';

    public function __construct( $warna = '', $tipe = ''){
        $this->warna = $warna;
        $this->tipe = $tipe;
    }

    public function setwarna($warna){
        $this->warna = $warna;
    }

    public function settipe($tipe){
        $this->tipe = $tipe;
    }


    public function getwarna(){
        return $this->warna;
    }

    public function gettipe(){
        return $this->tipe;
    }

    
}

?>